components/__tests__/Template-test.js

```diff
@@ -37,7 +37,7 @@ describe('Template', () => {
     renderer.render(<Template {...props} />);
     let out = renderer.getRenderOutput();
 
-    expect(out).toEqualJSX(<div dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
+    expect(out).toEqualJSX(<div className={undefined} dangerouslySetInnerHTML={{__html: 'it works with strings'}}></div>);
```

Comment:
Yes I was too fast on this, @redox there's no "cssClass" prop on template (also present in documentation). SO I wonder what are we doing here?